scripts/core/editor3/store/index.ts

```diff
@@ -94,6 +95,7 @@ export interface IEditorStore {
     abbreviations: any;
     loading: boolean;
     limitConfig?: EditorLimit;
+    softLimitConfig?: EditorLimit;
```

Comment:
you can't use `EditorLimit` type here since `CharacterLimitUiBehavior` config option doesn't exist for soft limit.